CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## v1.6.0 / 2022-01-28
```

Comment:
Once again, this date is made up and will need to be settled before merging...

Reply:
yes, you may want to do the release as a separate commit/PR, and just add to the unreleased section in this PR

there may be other things in the 1.6.0 release